examples/nidmm_measurement/README.md

```diff
@@ -24,5 +24,12 @@ This is a MeasurementLink example that performs a measurement using an NI DMM.
 
 This example requires an NI DMM (e.g. PXIe-4081).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```

Comment:
Did you notice that the rest of the content in these files is word wrapped? Please word-wrap the new content so it looks nice in a text editor. I have been using the Rewrap extension for VS Code: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Reply:
Which column have you been using for wrapping in .md files?

Reply:
I wrapped with column 100.

Reply:
I've been using the default.

Reply:
I rewrapped the files using the default setting, which uses about 80 columns.